lib/airrecord/table.rb

```diff
@@ -186,17 +186,18 @@ def destroy
 
     def serializable_fields(fields = self.fields)
       Hash[fields.map { |(key, value)|
-        if association(key)
-          assocs = value.map { |assoc|
-            assoc.respond_to?(:id) ? assoc.id : assoc
-          }
-          [key, assocs]
-        else
-          [key, value]
-        end
-      }]
-    end
-
+             if association(key)
+               value = [value] unless value.respond_to?(:map)
```

Comment:
I wasn't checking for an Array specifically. I was unsure if Hashes will also be passed in. Basically any enumerable. I probably should just check for an enumerable in that case.